File: sem1_cs/AnmolSingh_225HSBB004/Assignment13_03_12_22/c/q2.c
/*Write a program to count the number of characters in a file*/

#include<stdio.h>

int main(){
	FILE *fptr;
	char ch;
	int cntr, cntr2;
	fptr=fopen("1.txt", "r");
	cntr=0;
	cntr2=0;
	while(!feof(fptr))
	{
		fscanf(fptr,"%c",&ch);
		if((ch>=65 && ch<=90) || (ch>=97&& ch<=122)){cntr2++;}
		cntr++;
	}

	printf("The number of characters in the file is %d.\n No. of alphabets in the file is %d\n", cntr, cntr2);

	return (0);
}

File: sem1_cs/AnmolSingh_225HSBB004/Assignment13_03_12_22/python/q2.py
#!/usr/bin/python3
#Write a program to count the number of characters in a file

fp = open("q1.txt","w")
print("Enter 5 lines into the file:")
for i in range(5):
    line=input()
    fp.write(line+'\n')
fp.close()
cnt=0
letters=0

fp = open("q1.txt","r")
file=fp.read()
for char in file:
    asci=ord(char)
    cnt+=1
    if((asci>=65 and asci<=90) or (asci>=97 and asci<=122)):
        letters+=1
fp.close()
    
print("The number of characters in the file is", cnt,".\nNo. of alphabets in the file is",letters)